scripts/tokenomics-emulation/network.ts

```diff
@@ -110,6 +120,34 @@ class Network {
     });
   }
 
+  /**
+   * Calls the report_metrics function in cere02 smart contract
+   * @param sender Sender
+   * @param dataRec Data Rec
+   * @param dataRep Data Rep
+   * @param reqRec Request Rec
+   * @param reqRep Request Rep
+   * @returns hash
+   */
+  public async ddcReportMetrics(sender: KeyringPair, dataRec: string, dataRep: string, reqRec: string, reqRep: string) {
```

Comment:
Move to a separate class `DdcSmartContract`.

Reply:
Moved to new Class